src/components/Dashboard/TransactionModal/Transaction.module.css

```diff
@@ -0,0 +1,314 @@
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.modalContent {
+  position: relative;
+  background-color: #fff;
+  width: 320px;
+  padding: 0px;
+  border-radius: 8px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.15);
+  text-align: -webkit-center;
+}
+
+.modalTitle {
+  font-family: 'Poppins';
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 36px;
+  margin: 0px;
+  margin-top: 20px;
```

Comment:
Te wszystkie px są traktowane jako zła praktyka, tam gdzie nie potrzebujesz `TWARDO` zdefiniowanych wartości (czyli prawie nigdzie nie potrzebujesz poza breakpointami w @media) ustawiaj jednosktki relatywne (rem, em, % itd. Jak nie jesteś pewien co dać to wal rem) 
![image](https://github.com/KitsuneAkvma/wallet/assets/106407746/4b37f9f6-18ee-4428-a3f4-a429db1b7036)
Polecam to rozszerzenie

Reply:
Dodatkowo w `border-radius`, `box-shadow`, pozycjonowaniu absolutnym można używać px bez problemu